### PR TITLE
Thomas/dashboards

### PR DESCRIPTION
**Pull request by anagrius**

### CODE REVIEW

File: content/dashboards/embedding-widgets.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+You can embed a single Humio dashboard widget into an external HTML site using an iframe or in other media.
+
+The feature relies on the security model of [Shared Secret URLs]({{ ref "sharing-dashboards.md" }}),
+and you can only embed widgets that are accessible with a shared secret URL.
+
+## Embedding a widget in an iframe
+
+1. Add the widget you want to share to a dashboard.
+
+Tip: You can create a special dashboard called e.g. "Embedded Widgets" and add all the widgets you
```
Comment (mwl) [inline content/dashboards/embedding-widgets.md]: Shouldn't we be using a `notice` shortcode?

Comment (anagrius) [reply content/dashboards/embedding-widgets.md]: yes, maybe

File: content/dashboards/adding-a-widget-to-a-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (pmech) [inline content/dashboards/adding-a-widget-to-a-dashboard.md]: I could use a screen shot on this page

File: content/dashboards/parameters.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+Parameters are added to the query using the `?parameterName` syntax.
+
+## Creating a parameterized dashboard
+
+Lets say that we want to create a dashboard that monitors a set of servers and that
+those servers are placed in racks and run in different availability zones.
+
+Assuming that log entries are attributed with the following fields:
+
+`host` - The hostname of the server.
```
Comment (pmech) [inline content/dashboards/parameters.md]: Put these in a table for easier reading

File: content/dashboards/parameters.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+```humio
+host = ?host and rackId = ?rackId and zoneId=?zoneId | timechart(loglevel)
+```
+
+The query contains three parameters: `?host`, `?rackId` and `?zoneId`.
+
+Running the query produces a time chart widget with one series per log level. This allows
+us to see the activity level on servers and e.g. how many errors occur.
+
+![Parameters in a Query](/images/pages/dashboards/parameters-in-query.png)
```
Comment (pmech) [inline content/dashboards/parameters.md]: broken link locally

File: content/dashboards/sharing-dashboards.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+A shared secret URL is a URL containing a special authentication token that grants read-only
+access to anyone that has the link.
+
+## Creating a Shared Secret URL
+
+1. Visit the dashboard you want to make a read-only link for
+
+2. Click the "More" menu item (icon with three dots).
+
+![The more menu](/images/pages/dashboards/more-menu.png)
```
Comment (pmech) [inline content/dashboards/sharing-dashboards.md]: broken link locally

**Pull request merged**